src/main/java/seedu/moneygowhere/model/spending/SpendingList.java

```diff
@@ -8,22 +8,13 @@
 
 import javafx.collections.FXCollections;
 import javafx.collections.ObservableList;
-import seedu.moneygowhere.model.spending.exceptions.DuplicateSpendingException;
 import seedu.moneygowhere.model.spending.exceptions.SpendingNotFoundException;
 
 /**
- * A list of spendings that enforces uniqueness between its elements and does not allow nulls.
- * A Spending is considered unique by comparing using {@code Spending#isSameSpending(Spending)}.
- * As such, adding and updating of spending uses Spending#isSameSpending(Spending) for equality so as to ensure that the
- * Spending being added or updated is unique in terms of identity in the UniqueSpendingList.
- * However, the removal of a Spending uses Spending#equals(Object) so
- * as to ensure that the Spending with exactly the same fields will be removed.
```

Comment:
Could removing this have any side effects when removing an existing spending?

Reply:
nope